charts/rasa-x/templates/rasa-deployments.yaml

```diff
@@ -100,7 +100,7 @@ spec:
           value: "{{ .rasaEnvironment }}"
         - name: "RASA_MODEL_SERVER"
           value: "http://{{ include "rasa-x.host" $ }}:{{ $.Values.rasax.port }}/api/projects/default/models/tags/{{ .modelTag }}"
-
+        {{- include "rasa.extra.envs" $ | nindent 8 }}
```

Comment:
I think we should have separate `extraEnvs` for `rasa` and `rasa-x`.Their environment variables might be relatively similar now, but they might diverge more in the future.

Reply:
Yes.  I thought of this and an initial version had them as separate.  I collapsed them as a simplification but I agree it's probably better in the long run for them to be allowed to be defined as a separate set of extra env vars.